src/main/java/org/gridsuite/study/server/NotificationService.java

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server;
+
+import org.gridsuite.study.server.networkmodificationtree.dto.InsertMode;
+import org.gridsuite.study.server.utils.annotations.PostCompletion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.gridsuite.study.server.StudyService.*;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com
+ */
+@Service
+public class NotificationService {
+
+    static final String HEADER_USER_ID = "userId";
+    static final String HEADER_STUDY_UUID = "studyUuid";
+    static final String HEADER_NODE = "node";
+    static final String HEADER_UPDATE_TYPE = "updateType";
+    static final String UPDATE_TYPE_STUDIES = "studies";
+    static final String UPDATE_TYPE_STUDY_DELETE = "deleteStudy";
+    static final String HEADER_ERROR = "error";
+    static final String HEADER_UPDATE_TYPE_SUBSTATIONS_IDS = "substationsIds";
+    static final String HEADER_UPDATE_TYPE_DELETED_EQUIPMENT_ID = "deletedEquipmentId";
+    static final String HEADER_UPDATE_TYPE_DELETED_EQUIPMENT_TYPE = "deletedEquipmentType";
+
+    static final String HEADER_NODES = "nodes";
```

Comment:
Use an enum for notification types ?

Reply:
Using enum for notification types would add complexity to the code. Since we only need String to pass as parameters to the MessageBuilder (no value checking needed), static final String seem sufficient.